cmd/cluster-watchall/main.go

```diff
@@ -91,7 +91,10 @@ func main() {
 	cliOpts := NewClientOpts()
 	cliOpts.AddFlags(fs)
 	fs.Var(&utilflag.IPPortVar{Val: &serverBindAddress}, "server-bind-address", "The IP address with port at which to serve /metrics and /debug/pprof/")
-	fs.Parse(os.Args[1:])
+	includeSubresources := false
+	fs.BoolVar(&includeSubresources, "include-subresources", includeSubresources, "list subresources too")
+	args := os.Args[1:]
+	fs.Parse(args)
 
```

Comment:
why adding the args tmp variable?

Reply:
To help debugging a mystery that I am having about command line parsing.